src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const longTermDays = 7;
+  const longTermDiscount = 50;
+  const shortTermDays = 3;
+  const shortTermDiscount = 20;
+  let totalCost = days * dailyRate;
+
+  if (days >= longTermDays) {
+    totalCost = totalCost - longTermDiscount;
```

Comment:
here and in another `if` you can immediately return result (e.g `totalCost - longTermDiscount`) so you won't need the `else` statement and without changing totalCost so it can be `const`